### PR TITLE
Updated the extended FSiAdminBundle form theme template

### PR DESCRIPTION
**Pull request by szymach**

### CODE REVIEW

**Comment by rn0** [comment]
:+1:

**Comment by szymach** [comment]
@fsi-open

File: .travis.yml [hunk 1]
```diff
@@ -29,20 +29,22 @@ before_install:
...
     - sleep 5
     - mkdir -p features/fixtures/project/web/bundles/fsiadmin
     - mkdir -p features/fixtures/project/web/bundles/fsiadmintranslatable
+    - mkdir -p features/fixtures/project/web/bundles/ivoryckeditor
     - cp Resources/public/* features/fixtures/project/web/bundles/fsiadmintranslatable
 
 script:
-    - if [ -d vendor/fsi/admin-bundle/Resources/public ]; then cp vendor/fsi/admin-bundle/Resources/public/* features/fixtures/project/web/bundles/fsiadmin; fi
+    - if [ -d vendor/fsi/admin-bundle/Resources/public ]; then cp -r vendor/fsi/admin-bundle/Resources/public/* features/fixtures/project/web/bundles/fsiadmin; fi
+    - if [ -d vendor/egeloen/ckeditor-bundle/Resources/public ]; then cp -r vendor/egeloen/ckeditor-bundle/Resources/public/* features/fixtures/project/web/bundles/ivoryckeditor/; fi
```
Comment (chives) [inline .travis.yml]: please install assets like here - https://github.com/fsi-open/admin-bundle/blob/master/.travis.yml#L40

**Comment by szymach** [comment]
@fsi-open

**Comment by chives** [comment]
:+1:

**Pull request merged**